docs/whats-new/2020-05.md

```diff
@@ -0,0 +1,155 @@
+
+## C# language
+
+### New articles
+
+- [-nullable (C# Compiler Options)](/dotnet/csharp/language-reference/compiler-options/nullable-compiler-option)
+  - Adding -nullable C# compiler option details
+  - Adding -nullable C# compiler option details
+
+## .NET Framework
+
+### New articles
+
+- [ComNetOS class](/dotnet/framework/additional-apis/system.net.comnetos)
+  - Add System.Net APIs (1 of 3)
+  - Add System.Net APIs (1 of 3)
+- [ExceptionHelper class](/dotnet/framework/additional-apis/system.net.exceptionhelper)
+  - Add System.Net APIs (1 of 3)
+  - Add System.Net APIs (1 of 3)
+- [HttpStatusDescription class](/dotnet/framework/additional-apis/system.net.httpstatusdescription)
+  - Add System.Net APIs (1 of 3)
+  - Add System.Net APIs (1 of 3)
+- [Logging class](/dotnet/framework/additional-apis/system.net.logging)
+  - Add System.Net APIs (1 of 3)
+  - Add System.Net APIs (1 of 3)
+- [MailAddressParser class](/dotnet/framework/additional-apis/system.net.mail.mailaddressparser)
+  - Add System.Net APIs (2 of 3)
+  - Add System.Net APIs (2 of 3)
+- [QuotedPairReader class](/dotnet/framework/additional-apis/system.net.mail.quotedpairreader)
+  - Add System.Net APIs (2 of 3)
+  - Add System.Net APIs (2 of 3)
+- [MailBnfHelper class](/dotnet/framework/additional-apis/system.net.mime.mailbnfhelper)
+  - Add System.Net APIs (3 of 3)
+  - Add System.Net APIs (3 of 3)
+- [ServicePointManager.CloseConnectionGroups method](/dotnet/framework/additional-apis/system.net.servicepointmanager.closeconnectiongroups)
+  - Add System.Net APIs (2 of 3)
+  - Add System.Net APIs (2 of 3)
+- [UnsafeNclNativeMethods class](/dotnet/framework/additional-apis/system.net.unsafenclnativemethods)
+  - Add System.Net APIs (2 of 3)
+  - Add System.Net APIs (2 of 3)
+- [WebHeaderCollection.AddInternal method](/dotnet/framework/additional-apis/system.net.webheadercollection.addinternal)
+  - Add System.Net APIs (2 of 3)
+  - Add System.Net APIs (2 of 3)
+
+## F# language
+
+### Updated articles
+
+- [Keyword Reference](/dotnet/fsharp/language-reference/keyword-reference)
+  - Fix default checksum algorithm
+  - Fix default checksum algorithm
+
+## ML.NET
+
+### Updated articles
+
+- [The ML.NET CLI command reference](/dotnet/machine-learning/reference/ml-net-cli-reference)
+  - Update mlnet cli docs
+  - Update mlnet cli docs
+
+## .NET for Apache Spark
+
+### New articles
+
+- [Use broadcast variables in .NET for Apache Spark](/dotnet/spark/how-to-guides/broadcast-guide) - Added docs for broadcast and UDF guide
+- [Create user-defined functions (UDF) in .NET for Apache Spark](/dotnet/spark/how-to-guides/udf-guide) - Added docs for broadcast and UDF guide
+
+## .NET
+
+### New articles
+
+- [Implement a DisposeAsync method](/dotnet/standard/garbage-collection/implementing-disposeasync) - Implement the async dispose pattern
+
+## Community contributors

```

Comment:
The generated list contains *all* the contributors, not only ones from community. Isn't that automated yet or the section should be renamed?